Sources_v3/StreamChatUI/ChatChannel/MessageComposerInputAccessoryViewController.swift

```diff
@@ -222,14 +233,17 @@ open class MessageComposerInputAccessoryViewController<ExtraData: ExtraDataTypes
     
     // MARK: Attachments
     
-    var imageAttachments: [UIImage] = [] {
+    var imageAttachments: [URL] = [] {
         didSet {
             didUpdateImageAttachments()
         }
     }
     
     func didUpdateImageAttachments() {
-        composerView.attachmentsView.previews = imageAttachments
+        composerView.attachmentsView.previews = imageAttachments.compactMap {
+            guard let data = try? Data(contentsOf: $0) else { return nil }
+            return UIImage(data: data)
+        }
```

Comment:
Out of scope of this PR (cc @mantergo) -> please change the logic here such that we work with the local URL only. Let's leave the work of loading the image data to Nuke.

Reply:
removed composer changes from the PR 👍